### PR TITLE
Add `Electronic WeChat.app` v1.2.0.

### PR DESCRIPTION
**Pull request by zhongyi-tong**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

You can find the app here:
https://github.com/geeeeeeeeek/electronic-wechat

### CODE REVIEW

File: Casks/electronic-wechat.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'electronic-wechat' do
+  version '1.2.0'
+  sha256 :no_check
```
Comment (adidalal) [inline Casks/electronic-wechat.rb]: needs this set

File: Casks/electronic-wechat.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+  url 'https://github.com/geeeeeeeeek/electronic-wechat/releases/download/v1.2.0/mac-osx.tar.gz'
```
Comment (adidalal) [inline Casks/electronic-wechat.rb]: use url versioning

Comment (adidalal) [inline Casks/electronic-wechat.rb]: You can add an appcast

**Comment by geeeeeeeeek** [comment]
OK, will do.

**Comment by vitorgalvao** [comment]
You haven’t in fact ran `brew cask style --fix {{cask_file}}` after making all changes. Travis found an error.

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in b4b67bd999339d504ebeb49c6250eadd4f934ad1. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by geeeeeeeeek** [comment]
@adidalal One question. Why the app is linked to `~/Applications` instead of `/Applications`?

**Comment by adidalal** [comment]
It's a config variable you probably have set. Check homebre_cask_opts

**Pull request closed without merging**